SpatialGDK/Source/SpatialGDK/Public/Interop/SpatialStrategySystem.h

```diff
@@ -39,12 +39,12 @@ struct FStrategySystemViews
 class FSpatialStrategySystem
 {
 public:
-	FSpatialStrategySystem(TUniquePtr<FPartitionManager> InPartitionMgr, FStrategySystemViews InViews,
-						   TUniquePtr<FLoadBalancingStrategy> Strategy);
+	FSpatialStrategySystem(FStrategySystemViews InViews, TUniquePtr<FLoadBalancingStrategy> Strategy,
```

Comment:
The In prefix makes me think these should be const?

Reply:
Nah, constructed externally from other objects in the NetDriver and passed in to the SS.